src/Random.cpp

```diff
@@ -85,13 +88,43 @@ namespace galsim {
         clearCache();
     }
 
+    // Next two functions shamelessly stolen from
+    // http://stackoverflow.com/questions/236129/split-a-string-in-c
+    std::vector<std::string> &split(const std::string &s, char delim,
```

Comment:
I normally wouldn't mention this, but since we're discussing style here, I generally prefer the space to be after the &.  The return type here is `std::vector<std::string>&`.  So it makes things clearer to connect the `&` there rather than to adjoin it to the function name.  Likewise with the argument types.  No need to change it though.  Just putting that out there...  :)